environmentManager/__tests__/environmentManager.test.js

```diff
@@ -0,0 +1,281 @@
+const request = require('supertest');
+const app = require('../../app');
+
+// Mocked client to return authorization key when getTreatment is called
+jest.mock('../../sdk', () => ({
+  getSplitFactory: jest.fn((settings) => {
+    let sdk = jest.requireActual('../../sdk');
+    const factory = sdk.getSplitFactory(settings);
+    const client = factory.client();
+    client.getTreatment = jest.fn(() => { return factory.settings.core.authorizationKey; });
+    client.getTreatments = jest.fn(() => { return [factory.settings.core.authorizationKey]; });
+    client.getTreatmentWithConfig = jest.fn(() => { return { treatment: factory.settings.core.authorizationKey, config: null }; });
+    client.getTreatmentsWithConfig = jest.fn(() => { return [{ treatment: factory.settings.core.authorizationKey, config: null }]; });
+    client.track = jest.fn(() => {});
+    client.destroy = jest.fn(() => {});
+    return factory;
+  }),
+}));
+
+// Multiple environment
+// Mocked client should return authorizationKey when getTreatment is called
+// to verify that environmentManager is maping the right one for each authToken
+describe('environmentManager', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+
+
+
+  // getTreatment
+  describe('get-treatment', () => {
+    // Testing environment for authToken test-multiple
+    test('[GET] should be 200 if is valid authToken and return testapikey', async (done) => {
```

Comment:
If you are using `async-await` syntax, you don't need to use `done`.
Example: 
```
    test('[GET] should be 200 if is valid authToken and return testapikey', async () => {
      const response = await request(app)
        .get('/client/get-treatment?key=test&split-name=multiple-environment')
        .set('Authorization', 'test-multiple');
      expect(response.body.treatment).toEqual('testapikey');
    });
```

Check/update in the others tests